### ISSUE TITLE
ArchLinux + 9p fails on GitHub Actions (but not on local MacBook): `failed to get unprivileged mount flags for \"/home/runner/lima-container-engine-test-tmp/random\": stat /home/runner/lima-container-engine-test-tmp/random: no such file or directory`

### ISSUE THREAD

**Issue by AkihiroSuda** [lima-vm/lima#2701]
```
level=fatal msg="failed to get unprivileged mount flags for \"/home/runner/lima-container-engine-test-tmp/random\": stat /home/runner/lima-container-engine-test-tmp/random: no such file or directory"
```

https://github.com/lima-vm/lima/pull/1953/files/88cca7543122b694b7392e25816676779b9f38a3#r1790627760

**Comment by AkihiroSuda** [comment]
> Looks like we are hitting the kernel bug introduced in v6.9-rc1, fixed in v6.12-rc5: `Revert "fs/9p: simplify iget to remove unnecessary paths"` https://github.com/torvalds/linux/commit/be2ca38

_Originally posted by @AkihiroSuda in https://github.com/lima-vm/lima/issues/2821#issuecomment-2447803822_

**Comment by AkihiroSuda** [comment]
Carried as:
- https://github.com/lima-vm/lima/pull/2822

**Comment by jandubois** [comment]
I think this issue is "fixed" by #2822.
